### DISCUSSION TITLE
Addition of SECURITY.md ?

### DISCUSSION THREAD

**Discussion by andy778** [schemathesis/schemathesis#2508] [Ideas]
Have you been thinking of adding a SECURITY.md ?  

https://github.com/ossf/scorecard/blob/main/docs/checks.md#security-policy, Current score is "Aggregate score: 5.8 / 10"

**Comment by Stranger6667** [comment]
I've put together some basic [SECURITY.md](https://github.com/schemathesis/schemathesis/blob/master/SECURITY.md), feel free to suggest improvements if anything is missing.
